ENIGMAsystem/SHELL/Bridges/SDL-OpenGL/graphics_bridge.cpp

```diff
@@ -55,14 +55,16 @@ void ScreenRefresh() {
   SDL_GL_SwapWindow(windowHandle);
 }
 
-}
+} // namespace enigma
 
 namespace enigma_user {
-  void set_synchronization(bool enable) {
-    SDL_GL_SetSwapInterval(enable);
-  }
 
-  void display_reset(int samples, bool vsync) {
-    set_synchronization(vsync);
-  }
+void set_synchronization(bool enable) {
```

Comment:
Admittedly not the time to bring this up, but... is this a GM function? Why doesn't it start with `draw` or `graphics`?

Reply:
Yes it is, it is obsolete in Studio (deprecated in favour of display_reset() which has a vsync bool argument) and it dates back to at least 8.0 if not earlier. Not sure why Mark Overmars did not keep coherent standard.

Reply:
Thanks for helping with my review Banshee! Yeah Josh, he's correct.